client/modules/User/components/SignupForm.jsx

```diff
@@ -157,16 +169,14 @@ function SignupForm() {
                     <input
                       className="form__input"
                       type={showConfirmPassword ? 'text' : 'password'}
-                      aria-label={t('SignupForm.ConfirmPasswordARIA')}
```

Comment:
The label here, the `aria-hidden` on line 169, and `aria-live` on line 179 should also be added back!